crates/sp-domains-fraud-proof/src/fraud_proof.rs

```diff
@@ -527,12 +527,16 @@ pub enum InvalidBundlesProofData<Number, Hash, MmrHash, DomainHeader: HeaderT> {
     },
 }
 
+/// A proof about a bundle that was marked invalid (but might or might not actually be invalid).
 #[derive(Debug, Decode, Encode, TypeInfo, PartialEq, Eq, Clone)]
 pub struct InvalidBundlesProof<Number, Hash, MmrHash, DomainHeader: HeaderT> {
     pub bundle_index: u32,
+    /// The invalid bundle type that the bundle was marked with.
     pub invalid_bundle_type: InvalidBundleType,
-    pub is_true_invalid_fraud_proof: bool,
-    /// Proof data of the invalid bundle
+    /// If `true`, the fraud proof must prove the bundle was correctly marked invalid.
+    /// If `false`, it must prove the bundle was marked invalid, but is actually valid.
+    pub is_good_invalid_fraud_proof: bool,
```

Comment:
Honestly, I would prefer the previous naming itself.
Here is my thought process
- `is_true_invalid_fraud_proof` signifies whether fraud proof is targetting a truly invalid bundle or  if false signifies if targetting a proof to show incorrectly marked invalid bundle.

When it comes to `is_good_invalid_fraud_proof`, I'm not sure, without context, understand what good signifies here unlike TrueInvalid which brings more context self contained.

It could be that my grammer maybe off here. 
I do not want to block this PR on this name change but in the past there was a similar dicussion and final result ended up being going with True Prefix since the variants in it are just 2 and easily understood

Reply:
Oops, looks like this already merged.

Yep, naming things is hard, particularly when we have multiple layers of negatives: “fraud”, “invalid”, and then the `bool` possibly being `false`.

It might be there is no perfect solution here, happy to have it changed back to “truly” (which is different enough from a `true` `bool` to make it a bit easier).